src/components/Calculator.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
+  render() {
+    return (
+      <div className="MainBody">
+        <div className="result">0</div>
+        <div className="num-symb">
+          <button className="buttons" type="button">AC</button>
+          <button className="buttons" type="button">+/-</button>
+          <button className="buttons" type="button">%</button>
+          <button className="buttons operator" type="button">÷</button>
+        </div>
+        <div className="num-symb">
+          <button className="buttons" type="button">7</button>
+          <button className="buttons" type="button">8</button>
+          <button className="buttons" type="button">9</button>
+          <button className="buttons operator" type="button">×</button>
+        </div>
+        <div className="num-symb">
+          <button className="buttons" type="button">4</button>
+          <button className="buttons" type="button">5</button>
+          <button className="buttons" type="button">6</button>
+          <button className="buttons operator" type="button">-</button>
+        </div>
+        <div className="num-symb">
+          <button className="buttons" type="button">1</button>
+          <button className="buttons" type="button">2</button>
+          <button className="buttons" type="button">3</button>
+          <button className="buttons operator" type="button">+</button>
+        </div>
+        <div className="num-symb">
+          <button className="buttons" id="bigbtn" type="button">0</button>
+          <button className="buttons" type="button">.</button>
+          <button className="buttons operator" type="button">=</button>
```

Comment:
- As the project requests the use of props, you will need to have at least one child component. :thinking: 
For that reason, I suggest creating a Button component that will be called every time you need it. :recycle: 

- For this milestone, the calculator must be working properly, kindly make sure it works by creating a function that handles clicks on button. :recycle:

Reply:
Hey @GedeonTS  
Kindly can you elaborate what it means by the calculator should be working properly? I think its in the next milestone that we should include the logic.
Thanks in advance